src/api/c/approx.cpp

```diff
@@ -86,24 +86,29 @@ af_err af_approx1_uniform(af_array *yo, const af_array yi,
             return af_create_handle(yo, 0, nullptr, yi_info.getType());
```

Comment:
I think even the following can be refactored.
`af_create_handle(yo, 0, nullptr, yi_info.getType());`
to the following
`createHandle(dim4(0,0,0,0), yi_info.getType());`
Please test this.